monai/utils/sliding_window_inference.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import math
+import numpy as np
+import torch
+from monai.data.transforms import ImageEndPadder
+
+
+def sliding_window_inference(inputs, roi_size, sw_batch_size, predictor, device):
+    """Use SlidingWindow method to execute inference.
+
+    Args:
+        roi_size (list, tuple): the window size to execute SlidingWindow inference.
+        sw_batch_size (int): the batch size to run window slices.
+        predictor: a portion from numpy.lib.arraypad.pad is copied below.
```

Comment:
Does this comment describe predictor correctly?

Reply:
Hi @ericspod ,

Thanks for your review! That's a typo, I will fix here or in @wyli 's additional PR directly.